image_width: 320
image_height: 240


narrow_stereo:
  camera_matrix:
    rows: 3
    cols: 3
    data: [158.704340, 0.000000, 144.205532, 0.000000, 157.697764, 110.304166, 0.000000, 0.000000, 1.000000]
  distortion_coefficients:
    rows: 1
    cols: 5
    data: [-0.282143, 0.058551, -0.000493, -0.001844, 0.000000]
  rectification_matrix:
    rows: 3
    cols: 3
    data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
  projection_matrix:
    rows: 3
    cols: 4
    data: [110.171806, 0.000000, 142.530551, 0.000000, 0.000000, 121.773666, 103.104575, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
